documentation/sdk/docs/how_tos/native_tokens/send.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Send Native Tokens
+sidebar_label: Send
+description: 'How to send native tokens'
+image: /img/logo/iota_mark_light.png
+keywords:
+- how to
+- send native token
+- send
+- native token
+- nodejs
+- python
+- rust
+---
+
+import CodeBlock from '@theme/CodeBlock';
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import NodejsCode from '!!raw-loader!../../../../../bindings/nodejs/examples/how_tos/native_tokens/send.ts';
+import PythonCode from '!!raw-loader!../../../../../bindings/python/examples/how_tos/native_tokens/send.py';
+import RustCode from '!!raw-loader!../../../../../sdk/examples/how_tos/native_tokens/send.rs';
+
+After you have [minted](mint.mdx) a 
+[native token](https://wiki.iota.org/shimmer/introduction/explanations/what_is_stardust/tokenization#native-tokens),
+you can easily send it by calling the `Account.send_native_tokens(params, options)` function.
+
+As with any output, you can set a
+[storage deposit](https://wiki.iota.org/shimmer/introduction/explanations/what_is_stardust/storage_deposit) and
```

Comment:
Storage deposit is required for every output and not something you can set and in the context of this example useless

Reply:
True that. I think the example was also changed 🤔